pkg/controllers/applicationfailover/crb_application_failover_controller.go

```diff
@@ -173,11 +172,6 @@ func (c *CRBApplicationFailoverController) updateBinding(ctx context.Context, bi
 	for _, cluster := range needEvictClusters {
 		allClusters.Delete(cluster)
 	}
-	if !features.FeatureGate.Enabled(features.GracefulEviction) {
-		for _, cluster := range needEvictClusters {
-			helper.EmitClusterEvictionEventForClusterResourceBinding(binding, cluster, c.EventRecorder, nil)
-		}
-	}
```

Comment:
In the same way, we might need to clean up these places:

https://github.com/karmada-io/karmada/blob/54be414d9f24eea643b74f4b7c1f5e294778dfb9/pkg/controllers/cluster/taint_manager.go#L193-L195

https://github.com/karmada-io/karmada/blob/54be414d9f24eea643b74f4b7c1f5e294778dfb9/pkg/controllers/cluster/taint_manager.go#L255-L257

Reply:
Updated as suggested.